libraries/radspreadprocessing/features/conditional-formatting.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Conditional Formatting
+page_title: Conditional Formatting
+slug: radspreadprocessing-features-conditional-formatting
+tags: conditional, format, values
+published: True
+position: 7
+---
+
+# Conditional Formatting
+
+
+The conditional formatting in spreadsheet documents helps you visualize the data inside in a user-friendly manner, making it easy to analyze data, find critical issues, patterns and trends.
+
+A conditional format changes the appearance of cells based on conditions that you specify. If the conditions are true, the cell range is formatted; if the conditions are false, the cell range is not formatted. There are many built-in conditions, and you can also create your own (including by using a formula that evaluates to True or False).
+
+This article describes the supported formatting rules, how to create and how to use them.
+
+## Rules
+
+SpreadProcessing supports many different types of formatting rules. Each rule implements the base **ConditionalFormattingRule** class. All of them are listed in **Table 1**.
+
+#### Table 1: Supported formatting rules
+
+| Rule | Description |
```

Comment:
I believe that it is better for the items in the first column to be bolded. The second column can be wider as well